### PR TITLE
docs(analytics-module-ga4): fix CSP documentation

### PR DESCRIPTION
**Pull request by NebojsaMrmak-TomTom**
## Hey, I just made a Pull Request!

The documentation for recently merged GA4 plugin seems to be copied over from the old GA plugin, however that won't wok out of the box. This PR fixes the documentation to include GA4 url in CSP.

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentatio
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by NebojsaMrmak-TomTom** [comment]
> ## Missing Changesets
> The following package(s) are changed by this PR but do not have a changeset:
> 
> * **@backstage/plugin-analytics-module-ga4**
> 
> See [CONTRIBUTING.md](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets) for more information about how to add changesets.
> 
> ## Changed Packages
> Package Name	Package Path	Changeset Bump	Current Version
> @backstage/plugin-analytics-module-ga4	plugins/analytics-module-ga4	**none**	`v0.1.0-next.0`

I did run `yarn changeset` but probably did something wrong. Can you help a first-time contributor? :)

File: plugins/analytics-module-ga4/README.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ backend:
   csp:
     connect-src: ["'self'", 'http:', 'https:']
     # Add these two lines below
-    script-src: ["'self'", "'unsafe-eval'", 'https://www.google-analytics.com']
-    img-src: ["'self'", 'data:', 'https://www.google-analytics.com']
+    script-src: ["'self'", "'unsafe-eval'", 'https://www.googletagmanager.com']
+    img-src: ["'self'", 'data:', 'https://https://www.googletagmanager.com']
```
Comment (adamdmharvey) [inline plugins/analytics-module-ga4/README.md:60]: Minor typo; Double protocol link, just FYI !

**Comment by clanghout** [comment]
> > ## Missing Changesets
> > The following package(s) are changed by this PR but do not have a changeset:
> > 
> > * **@backstage/plugin-analytics-module-ga4**
> > 
> > See [CONTRIBUTING.md](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md?rgh-link-date=2023-04-30T09%3A41%3A32Z#creating-changesets) for more information about how to add changesets.
> > ## Changed Packages
> > Package Name	Package Path	Changeset Bump	Current Version
> > @backstage/plugin-analytics-module-ga4	plugins/analytics-module-ga4	**none**	`v0.1.0-next.0`
> 
> I did run `yarn changeset` but probably did something wrong. Can you help a first-time contributor? :)

I think you need to select the module that changed in the interactive section that shows up when running the changeset command. Then you can type a short summary on why the changes are  made. I am not sure what the exact policy is on this, especially since this is a docs only change.

**Comment by jhaals** [comment]
This is a perfectly fine text to include in a `patch` changeset.

> Fixed documentation to include GA4 url in CSP config section

**Pull request closed without merging**